### PR TITLE
Remove the redundant example apps and packages

### PR DESCRIPTION
**Pull request by nikomakela**
The monorepo base was a clone of https://github.com/belgattitude/nextjs-monorepo-example. Now when the Hobbies-Helsinki is in a point that it's working with the example apps and packages, we can start cleaning the repo from redundant example apps and packages.

What packages we still need with Hobbies (and Tapahtumat and other monorepo apps that will be there in future) are:
- `components` for common global event related components
- `common-i18n` to serve translations
- `eslint-config-bases` for global linter configuration

### CODE REVIEW

File: .github/workflows/ci-hobbies-helsinki-check.yml [hunk 1]
```diff
@@ -76,7 +76,6 @@ jobs:
         run: |
           yarn install --immutable --inline-builds
         env:
-          PRISMA_SKIP_POSTINSTALL_GENERATE: true
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
```
Comment (nenonja) [inline .github/workflows/ci-hobbies-helsinki-check.yml:79]: playwright isnt used in hobbies, so all references to it could be removed from monorepo?

**Review by nenonja — COMMENTED** [review]
one comment, otherwise look good. Ci-packages check at least failed too, not sure if related to changes.

**Comment by nikomakela** [comment]
It seems like the components used the core-package as a dependency for the storybook stories.

- [ ] Remove the core-dependency from the components-package (stories) and fix the CI-packages test

File: .github/workflows/ci-hobbies-helsinki-check.yml (hunk 1 above)
Comment (nikomakela) [reply .github/workflows/ci-hobbies-helsinki-check.yml]: Playwright relations removed

**Review by nenonja — APPROVED** [review]

**Pull request merged**